mmcv/cnn/bricks/transformer.py

```diff
@@ -612,21 +568,19 @@ class TransformerLayerSequence(BaseModule):
 
     def __init__(self, transformerlayers=None, num_layers=None, init_cfg=None):
         super(TransformerLayerSequence, self).__init__(init_cfg)
-        if isinstance(transformerlayers, ConfigDict):
+        if isinstance(transformerlayers, dict):
```

Comment:
Docstring should be modified.

Reply:
Actually, In most cases, it is a `ConfigDict` except for using default arguments, and `ConfigDict` is a subclass of `dict`.